routes/web.php

```diff
@@ -525,6 +525,7 @@
         $router->get('/', "HXLController@index");
         $router->get('/licenses', 'HXLLicensesController@index');
         $router->get('/tags', 'HXLTagsController@index');
+        $router->post('/metadata', 'HXLMetadataController@store');
```

Comment:
I don't quite understand why this is write only? Do we never expect to read metadata again?

Reply:
(answered in slack but adding here to not forget)
Right now, we don't expect to read metadata from an endpoint, so this was spec'd as write only.